sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Release History
 
 ## 1.0.0-beta.5 (2021-03-09)
-Updated `Azure.Communication.Chat` version.
-- Added identifier serialization in Chat
-- Updated the SDK to use `CommunicationIdentifier` changes everywhere instead of string Id
-- Removal of nullable references
-- Added unit test and live test coverage 
+
+### Breaking Changes
+
+- Added support for communication identifiers instead of raw strings.
+
+### Added
+
+- Removed support for nullable reference types.
```

Comment:
This should move to breaking change section

Reply:
moved